integration-tests/contracts/ethereum_keeper_contracts.go

```diff
@@ -275,6 +278,31 @@ func (v *EthereumKeeperRegistry) SetConfig(config KeeperRegistrySettings) error
 			return err
 		}
 		return v.client.ProcessTransaction(tx)
+	case ethereum.RegistryVersion_2_0:
+		state, err := v.registry2_0.GetState(&callOpts)
+		if err != nil {
+			return err
+		}
+
+		tx, err := v.registry2_0.SetConfig(txOpts, ethereum.Config2_0{
+			PaymentPremiumPPB:    config.PaymentPremiumPPB,
+			FlatFeeMicroLink:     config.FlatFeeMicroLINK,
+			BlockCountPerTurn:    config.BlockCountPerTurn,
+			CheckGasLimit:        config.CheckGasLimit,
+			StalenessSeconds:     config.StalenessSeconds,
+			GasCeilingMultiplier: config.GasCeilingMultiplier,
+			MinUpkeepSpend:       config.MinUpkeepSpend,
+			MaxPerformGas:        config.MaxPerformGas,
+			FallbackGasPrice:     config.FallbackGasPrice,
+			FallbackLinkPrice:    config.FallbackLinkPrice,
+			// Keep the transcoder and registrar same. They have separate setters
+			Transcoder: state.Config.Transcoder,
+			Registrar:  state.Config.Registrar,
+		})
```

Comment:
setConfig has completely changed in 2.0. Will need to follow steps similar to https://github.com/smartcontractkit/chainlink/blob/develop/core/scripts/chaincli/handler/keeper_deployer.go#L72-L195